src/core/Directus/Services/AssetService.php

```diff
@@ -333,6 +309,11 @@ public function load () {
         $ext = pathinfo($this->fileName, PATHINFO_EXTENSION);
         if (Thumbnail::isNonImageFormatSupported($ext)) {
             $content = Thumbnail::createImageFromNonImage($content);
+            // For non supported file format make default thumbnail with jpeg extension
```

Comment:
This sounds dangerous. Where is this based on? 

If someone tries resizing a PDF, you'll use the JPEG extension and see what happens?

We can't assume that everything we don't know the type of is JPEG